### PR TITLE
Test the MVC default serialization options

### PR DESCRIPTION
**Pull request by hyzx86**

### CODE REVIEW

File: test/OrchardCore.Tests.Modules/ModuleSample/Controllers/ApiController.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+[ApiController]
+public class ApiController : Controller
+{
+    [HttpPost, HttpGet]
+    [Route("~/moduleSample/tests/testCustomQuery")]
+    public async Task<IActionResult> TestCustomQueryAsync()
+    {
+        var result = new CustomQueryResult { Data = new { user = "Tom", age = 5 } };
```
Comment (hyzx86) [inline test/OrchardCore.Tests.Modules/ModuleSample/Controllers/ApiController.cs:18]: Hi @MikeAlhayek , I want to create an Api in this module to simulate the return result of a custom query, but I don't know why I can't access this endpoint in my tests and it keeps returning 404

Comment (MikeAlhayek) [reply test/OrchardCore.Tests.Modules/ModuleSample/Controllers/ApiController.cs:18]: try removing `~/` from the route value.

File: test/OrchardCore.Tests/Serializers/JsonSerializerTests.cs [hunk 2]
```diff
@@ -74,4 +76,16 @@ public async Task DefaultContentSerializer_SerializeAndDeserialize_UserWithUserL
...
     }
+
+    [Fact]
+    public async Task ValidateTheCustomQueryResultResponse()
+    { 
+        using var context = new BlogContext();
+        await context.InitializeAsync();
+        await context.RunRecipeAsync("TestQueryRecipe.json");
+
+        var fistResponse = await context.Client.GetAsync("moduleSample/tests/testCustomQuery");
```
[the remark this answers is not in the document]
Comment (MikeAlhayek) [reply test/OrchardCore.Tests/Serializers/JsonSerializerTests.cs:87]: I don't think you enabled the correct feature. The feature that you created has `TestCustomQueryFeature` id not `ModuleSample` check the value in your recipe

[the remark this answers is not in the document]
Comment (hyzx86) [reply test/OrchardCore.Tests/Serializers/JsonSerializerTests.cs]: I adjusted it as you said. But it still doesn't seem to be working as expected

**Pull request closed without merging**